src/app/files/file-explorer.js

```diff
@@ -350,7 +350,7 @@ fileExplorer.prototype.init = function () {
 fileExplorer.prototype.publishToGist = function () {
   modalDialogCustom.confirm(
     'Create a public gist',
-    'Are you sure you want to publish all your files anonymously as a public gist on github.com?',
+    'Are you sure you want to publish all your files in browser directory anonymously as a public gist on github.com?',
```

Comment:
we should say that this will not include folder

Reply:
I would say it's fine. we would implement this anyway, oder?
Wouldn't this be too long with folder stuff?

Reply:
well I don't really think we can implement that easily, gists does not support having folder.
Your sentence is ok but it should explicitly state that folders are not uploaded